### DISCUSSION TITLE
The currnt.json configuration of the i18n is invalid.

### DISCUSSION THREAD

**Discussion by Dong-Qiu-Long** [facebook/docusaurus#8655] [Q&A]
<img width="1302" alt="image" src="https://user-images.githubusercontent.com/70185294/218255132-1bb8040d-d287-42e9-b29d-1571174b620a.png">
<img width="1042" alt="image" src="https://user-images.githubusercontent.com/70185294/218255146-38da4b91-83b3-417c-93ef-038cf4261c72.png">
This.doc.xxx configuration does not take effect。
Do you support this configuration？
No answer can be found in the documentation, and no relevant cases can be found。

**Comment by slorber** [comment]
This is expected to work. 

We use this on our own website: `sidebar.docs.doc.Using Git`

And it is correctly localized: https://docusaurus.io/fr/docs/i18n/git

<img width="279" alt="CleanShot 2023-02-15 at 17 21 09@2x" src="https://user-images.githubusercontent.com/749374/219088175-9d758ffb-2283-4e72-8f23-433998f282f7.png">

---



Please create an issue and provide a reproduction case so that we can investigate

**Reply by Dong-Qiu-Long** [reply]
How do you guys do that? Could you please send me the link to your case code？

**Reply by slorber** [reply]
We use Crowdin for translations so the files are not found in Git but they are here:
https://crowdin.com/translate/docusaurus-v2/630/en-fr?filter=basic&value=0

Please create a repro with Git translations and I'll tell you what you do wrong
